sign_test.go

```diff
@@ -134,3 +135,86 @@ func TestSignRemoveHeaders(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestVerifyDSAWithSHA1(t *testing.T) {
```

Comment:
This should probably go in `verify_test.go`. Ideally, we would generate the key/sig at runtime, but I guess that isn't easy since DSA doesn't implement `crypto.Signer`

https://github.com/mastahyeti/cms/blob/119fef08a078a262d54a40366d1c3f7de2e9d3f1/main_test.go#L23-L41

Reply:
Yea, that test was in the wrong spot. I moved it to verify_test.go. I agree that ideally we would generate the key/sig at runtime, but the fact that DSA doesn't implement crypto.Signer makes this a really big task. From what I can tell, it would require overhauling fakeca so that it wouldn't require crypto.Signer.

Reply:
That, or make another type that wraps `crypto/dsa.PrivateKey` and implements `crypto.Signer`. I might look into that later, but it isn't necessary for this PR.